util/install.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+echo "Starting font installation process..."
+
+# Get target root directory
+if [[ $(uname) == 'Darwin' ]]; then
+    # MacOS
+    sys_share_dir="/Library"
```

Comment:
This is never used.

Reply:
I guess the code comes from my snippet.

That variable would be used if the script allows system-wide installation.
As it does not (yet) you are right it's unused/